Quotient/room.cpp

```diff
@@ -719,6 +719,16 @@ QList<RoomMember> Room::members() const {
     return members;
 }
 
+Q_INVOKABLE QList<RoomMember> Room::membersTyping() const
```

Comment:
```suggestion
QList<RoomMember> Room::membersTyping() const
```
`Q_INVOKABLE` has no effect here.